encryption/store.go

```diff
@@ -23,7 +23,7 @@ import (
 //    b1, u6/u7/u8    => <e6/e7/e8, k8>
 //    b2, u1          => <e1', k1'>
 //
-// Then the following lookups have outputs
+// then the following lookups have outputs.
```

Comment:
Shouldn't this end in `:`?

Reply:
Likely, wasnt sure if it likes anything but a dot.